package.json

```diff
@@ -117,6 +117,7 @@
     "prettier": "^3.2.5",
     "react": "^16.14.0",
     "react-dom": "^16.14.0",
+    "react-markdown": "^8.0.7",
```

Comment:
Do we really need this?

Reply:
Not really! I mostly added it because in Q we are gonna be using this so I wanted to validate it works as expected but I can replace it by something else and avoid adding a devDependency here